### ISSUE TITLE
Some question about  temperature parameter

### ISSUE THREAD

**Issue by wangxyustc** [BinhuiXie/SPCL#3]
Very interesting work. But I want to ask a question, temperature is an important hyperparameter in contrastive learning, I found that in SDCA work, the temperature is set to 100, I want to know if the temperature in SPCL is the same as the SDCA setting, and if there is anything about the difference Temperatures lead to related experiments with different results.

Looking forward to your reply.

Best Regards,
Qianmo

**Comment by BinhuiXie** [comment]
Hello, @QianMo-wangxiaoyu 

Thank you for your attention.

For SPCL, the temperature is also fixed at 100. Since the pixel-level contrast loss contains a large number of contrast pairs, we use a larger temperature to balance the scale. In addition, we are exploring to operate on a fraction of these pixels. 

BTW, for the PDD metric (https://github.com/BIT-DA/SDCA/issues/6), I will deal with it in my free time.

Best,
Binhui

**Comment by wangxyustc** [comment]
Thank you very much!
Best wishes to you

**Comment by QianMo-wangxiaoyu** [comment]
hello, can you upload the source-only pretrained model?
